Keda.Cosmosdb.Scaler/src/Services/CosmosDbTrigger.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Data.Common;
+using Microsoft.Azure.Documents.ChangeFeedProcessor;
+
+
+namespace Keda.Cosmosdb.Scaler.Services
+{
+    internal class CosmosDbTrigger

```

Comment:
Let's use improve this a bit here:
- Lease should be a type with 3 properties
- We should not use `DocDB` as a name here, rather use `CosmosDb`